### PR TITLE
Add Gitter link to footer

### PR DESCRIPTION
**Pull request by flyswatter**
Much Ethereum development and discussion happens on the Gitter chat, so I've added it to the Community section of the footer.

I think I matched the icon style pretty nicely:

screen shot 2016-10-05 at 10 46 44 am

### CODE REVIEW

**Comment by flyswatter** [comment]
Forgive my editor for automatically removing a lot of trailing whitespace, makes the diff look a lot more complicated than it is.

File: public/css/style.css [hunk 1]
```diff
@@ -2008,12 +2011,15 @@ footer {
 	.community ul li a.twitter {
 		background-image: url("../images/icons/icon-twitter@2x.png");
 	}
+  .community ul li a.gitter {
```
Comment (luclu) [inline public/css/style.css]: Please set the correct indent here.

File: views/includes/footer.jade [hunk 2]
```diff
@@ -53,13 +53,16 @@ footer.scrollme
...
+                    a(href='https://gitter.im/orgs/ethereum/rooms').gitter Gitter
+                li
                     a(href='http://ethereum.stackexchange.com').stackexchange Stack Exchange
                 li
                     a(href='https://www.facebook.com/ethereumproject').facebook Facebook                
                 li
                     a(href='http://ethereum.meetup.com').meetups Meetups
                 li
                     a(href='mailto:info@ethereum.org').email Email us
+
```
Comment (luclu) [inline views/includes/footer.jade]: And remove this line (not sure about jade's syntax, but it might break things).

**Comment by luclu** [comment]
@flyswatter great! Thank you!
I added some comments to your code.

**Comment by alexvandesande** [comment]
These changes (with the fixes requested) will be merged with https://github.com/ethereum/ethereum-org/pull/295/files

**Comment by flyswatter** [comment]
Thank you! I had forgotten about this, and appreciate that, Alex!

- Dan

**Pull request closed without merging**